org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/xpect/CannotModifyConstants.wlk.xt

```diff
@@ -4,21 +4,21 @@ class A {
 	const x = "sarasa"
 	
 	constructor() {
-		// XPECT errors --> "Cannot modify values" at "x"
+		// XPECT errors --> "Cannot modify constants" at "x"
 		x = 23
 	}
 	
 	method s() {
 		const aBlock = {a =>
-			// XPECT errors --> "Cannot modify values" at "a"
+			// XPECT errors --> "Cannot modify constants" at "a"
 			a = 23
 		}
-		// XPECT errors --> "Cannot modify values" at "aBlock"
+		// XPECT errors --> "Cannot modify constants" at "aBlock"
 		aBlock = null
 	}
 	
 	method xxx(aParam) {
-		// XPECT errors --> "Cannot modify values" at "aParam"
+		// XPECT errors --> "Cannot modify constants" at "aParam"
```

Comment:
Just to give you some feedback. "Young people" now use to create different branches to address 1 specific issue or task. Like smallers branches, to be quickly merged.
Instead of a big one with many commits or issues/tasks.
It makes it easier to review later, and should be more "agile".

This one is ok, but just letting you know about this other way of working